internal/service/event/main.go

```diff
@@ -0,0 +1,101 @@
+package event
+
+import (
+	"fmt"
+
+	"github.com/rarimo/geo-points-svc/internal/config"
+	"github.com/rarimo/geo-points-svc/internal/data"
+	"github.com/rarimo/geo-points-svc/internal/data/evtypes"
+	"github.com/rarimo/geo-points-svc/internal/data/evtypes/models"
+	"github.com/rarimo/geo-points-svc/internal/data/pg"
+	"github.com/rarimo/geo-points-svc/internal/service/handlers"
+)
+
+func Run(cfg config.Config, date int) error {
+	log := cfg.Log()
+	db := cfg.DB()
+	lvls := cfg.Levels()
+	evTypes := cfg.EventTypes()
+
+	balancesQ := pg.NewBalances(db)
+	eventsQ := pg.NewEvents(db)
+	referralsQ := pg.NewReferrals(db)
+
+	evType := evTypes.Get(models.TypeEarlyTest, evtypes.FilterInactive, evtypes.FilterByAutoClaim(true))
```

Comment:
You shouldn't filter by autoclaim here, because if event active, we can add fulfilled event and if autoclaim=true - claim.